Source/Frontend/UI/UI.csproj

```diff
@@ -428,6 +428,7 @@
     <Compile Include="UICore.cs" />
     <Compile Include="UIConnector.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Components\DataGridViewDraggable.cs" />
```

Comment:
Double-checking to make sure I chose the right spot for this.

Reply:
This is correct.